src/shared/services/fields/fields.utils.ts

```diff
@@ -155,11 +177,11 @@ export class FieldsUtils {
     }
   };
 
-  private static getMoneyGBP(fieldValue) {
+  private static getMoneyGBP(fieldValue: any): any {
```

Comment:
Just checking that there's no closer match than `any` for the `fieldValue` parameter and the function return type?

Reply:
I tried it as `string` but it was complaining so I reverted it to `any`. I'll have another look, though, as I could possibly change it to a composite type like `number | string`.

Reply:
The parameter _has_ to be `any` because that's what gets passed to the pipe. I've changed return type to `string` and added a comment to that effect.

Reply:
Ah, OK - that's fine